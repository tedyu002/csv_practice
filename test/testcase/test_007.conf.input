1997,Ford,E350,800000,TRUE
1991,Mercedes-Benz,S320,2800000,TRUE
2003,"Mercedes-Benz
2nd line,,, ,,,
3rd line ""ANGUS"",
4rd line """",,,,
5rd line",C220,1800000,TRUE
2016,Volvo,S60,1750000,FALSE
1991,Audi,A8,1850000,TRUE
2003,Mazda,CX-5,1150000,FALSE
2017,Skoda,Yeti,950000,FALSE
2017,Skoda,Rapid,958000,TRUE
2016,Skoda,Citigo,858000,FALSE
1991,Audi,Q7,2250000,TRUE
2016,Volvo,XC90,2550000,TRUE
2016,Volvo,V40,1550000,FALSE
2015,Volvo,S80,2350000,FALSE
2016,Volvo,XC60,2350000,FALSE
