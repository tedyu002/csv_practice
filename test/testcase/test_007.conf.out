FALSE,858000,"2016","Skoda","Citigo"
FALSE,950000,"2017","Skoda","Yeti"
FALSE,1150000,"2003","Mazda","CX-5"
FALSE,1550000,"2016","Volvo","V40"
FALSE,1750000,"2016","Volvo","S60"
FALSE,2350000,"2015","Volvo","S80"
FALSE,2350000,"2016","Volvo","XC60"
TRUE,800000,"1997","Ford","E350"
TRUE,958000,"2017","Skoda","Rapid"
TRUE,1800000,"2003","Mercedes-Benz2nd line,,, ,,,3rd line ""ANGUS"",4rd line """",,,,5rd line","C220"
TRUE,1850000,"1991","Audi","A8"
TRUE,2250000,"1991","Audi","Q7"
TRUE,2550000,"2016","Volvo","XC90"
TRUE,2800000,"1991","Mercedes-Benz","S320"
